apps/earn-protocol/app/portfolio/[walletAddress]/page.tsx

```
@@ -43,10 +58,48 @@ const PortfolioPage = async ({ params }: PortfolioPageProps) => {
+        await getPositionHistory({
+          network: vault.protocol.network,
+          address: walletAddress.toLowerCase(),
+          vault,
+        }),
+    ),
+  ).then((responses) =>
+    responses.reduce<{
+      [key: string]: GetPositionHistoryReturnType['positionHistory']
+    }>((acc, { positionHistory, vault }) => {
+      return {
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)